src/main/java/org/thoughtcrime/securesms/ConversationActivity.java

```diff
@@ -441,12 +441,13 @@ public boolean onPrepareOptionsMenu(Menu menu) {
       menu.findItem(R.id.menu_show_map).setVisible(false);
     }
 
-    if (!dcChat.canSend() || dcChat.isBroadcast() || dcChat.isMailingList()) {
+    if (!dcChat.canSend() || dcChat.isMailingList()) {
```

Comment:
`dcChat.isMailingList()` is `true` for incoming broadcasts (subscriber side) as of today? by looking at other parts where you introduced a IN_BROADCAST_CHANNEL chat type doesn't look like it to me, so here you would need to check if `isInBroadcastChannel()` and also hide the ephemeral timer option for them since they can't change the timer as a subscriber, right?

Reply:
It doesn't matter, `canSend()` is `false` for incoming brodcasts, anyways.